### PR TITLE
[Box] Atmospherics starts more set-up

### PR DESCRIPTION
**Pull request by ghost**
- All filters in Atmospherics were set to their maximum (4500 kPa). This saves everyone the bother of actually opening all of them to manually max them out
- All noxious gas tanks (CO2, Toxins, N2O) now start with output at 0 instead of default output. This means gas won't build up and be ready to release until someone changes the output value, preventing three-click plasma-floods
- The regular piping towards "External" (the two pump ports outside Atmospherics) was changed back to a volume pump. Having a pump here allows gas to build up to max out the pumps, instead of having barely 2000 kPa since the rest is being drained into the loop
- The pump separating the air output after it was separated from the mix input, and before the recycled input (basically the second last pump before outputting the air loop into distro) was removed. It serves absolutely no purpose, either recycled air mixes into distro or it doesn't

Those are precisely all the changes that were done in this PR

### CODE REVIEW

File: maps/tgstation.dmm [hunk 1]
```diff
@@ -6216,7 +6217,7 @@
...
 "cpD" = (/obj/machinery/atmospherics/pipe/simple/yellow/visible,/turf/simulated/floor,/area/engineering/atmos)
 "cpE" = (/obj/machinery/atmospherics/pipe/simple/cyan/visible,/turf/simulated/floor,/area/engineering/atmos)
-"cpF" = (/obj/machinery/computer/general_air_control/large_tank_control{frequency = 1441; input_tag = "n2o_in"; name = "Nitrous Oxide Supply Control"; output_tag = "n2o_out"; sensors = list("n2o_sensor" = "Tank")},/obj/machinery/atmospherics/pipe/simple/filtering/visible,/turf/simulated/floor{icon_state = "escape"; dir = 4},/area/engineering/atmos)
+"cpF" = (/obj/machinery/atmospherics/trinary/filter{dir = 1; filter_type = 4; icon_state = "intact_on"; name = "Gas filter (N2O tank)"; on = 1; target_pressure = 4500},/obj/machinery/camera{c_tag = "Atmospherics East"; dir = 8},/turf/simulated/floor{icon_state = "escape"; dir = 6},/area/engineering/atmos)
```
Comment (PJB3005) [inline maps/tgstation.dmm]: I TOLD YOU NOT TO MAKE MORE INSTANCES.

AAAAAAAAAAAAAAAAAAAAAAAAAAAAAH

**Comment by PJB3005** [comment]
While you're at it could you fix the pixel offset on the digital t valve before the filters?

File: maps/tgstation.dmm (hunk 1 above)
Comment (dylanstrategie) [inline maps/tgstation.dmm]: This is not a new instance. All filters are already instances because they are named. Same with the computers

Comment (PJB3005) [inline maps/tgstation.dmm]: Make subtypes then.

**Comment by N3X15** [comment]
There's a reason that we don't let you make balance decisions.

**Comment by PJB3005** [comment]
kek.

Also @N3X15 is on a PR closing spree

**Pull request closed without merging**